pyEpiabm/pyEpiabm/tests/test_unit/test_intervention/test_place_closure.py

```diff
@@ -0,0 +1,50 @@
+import unittest
+
+import pyEpiabm as pe
+from pyEpiabm.property import InfectionStatus
+from pyEpiabm.intervention import PlaceClosure
+from pyEpiabm.tests.test_unit.parameter_config_tests import TestPyEpiabm
+
+
+class TestPlaceClosure(TestPyEpiabm):
+    """Test the 'PlaceClosure' class.
+    """
+    @classmethod
+    def setUpClass(cls) -> None:
+        """Intialise a population with one infector and one
+        infectee, both in the same place and household.
```

Comment:
This is not what the code does. We create 2 people and assign them both to the same InfectionStatus (7). Do we need to create persons here @jiayuanz3 ?

Reply:
We only need one person that is infectious.